test_team_sci.py

```diff
@@ -1,8 +1,30 @@
 import unittest
-from pii_scan import show_aggie_pride, analyze_text
-
+from pii_scan import analyze_text, show_aggie_pride
 
 class TestTeamSci(unittest.TestCase):
     def test_aggie_pride(self):
         """Test to make sure the Aggie Pride function works"""
         self.assertEqual('Aggie Pride - Worldwide', show_aggie_pride())
+
+    def test_itin_detect(self):
+        """Test to make sure ITIN (US_ITIN) is detected"""
+
+        # Positive Testcase with a valid ITIN
+        results = analyze_text('My ITIN is 9XX-70-1234')
+        print(results)
+        self.assertNotIn('US_ITIN', str(results))
```

Comment:
This is not a positive test case - it's a negative one self.assertNotIn() checks that US_ITIN is not in the results string.  You want your positive test cases to use self.assertIn().  I would try something like this:
```
        # Positive Testcase with a valid ITIN
        results = analyze_text('My ITIN is 900701234')
        print(results)
        self.assertIn('US_ITIN', str(results))
```